consensus/core/src/test_dag_builder.rs

```diff
@@ -298,6 +346,7 @@ impl DagBuilder {
         blocks
     }
 
+    #[allow(dead_code)]
```

Comment:
This allow shouldn't be needed? If it's not used it should be removed
And if it's just temporary and you know it will be used again, then it's better to use `#[expect(dead_code)]` so one gets notified if it isn't dead code anymore

Reply:
ah, sorry I skipped this and merged it too quickly. We'll apply the suggestion in the next PR.